include/fizzy/fizzy.h

```diff
@@ -243,6 +243,15 @@ uint8_t* fizzy_get_instance_memory_data(FizzyInstance* instance);
 /// @note    Function returns memory size regardless of whether memory is exported or not.
 size_t fizzy_get_instance_memory_size(FizzyInstance* instance);
 
+/// Find exported table by name.
+///
+/// @param  instance        Pointer to instance.
+/// @param  name            The table name. NULL-terminated string. Cannot be NULL.
+/// @param  out_table       Pointer to output struct to store found table. Cannot be NULL.
+/// @returns                true if table was found, false otherwise.
+bool fizzy_find_exported_table(
```

Comment:
I was going to say why isn't this grouped with `find_exported_function`, but realised that is over the module. This still confuses me in our API, both C++ and in C.

Reply:
Yes, I put them separately, because this one operates on instance.

The change in #633 adds another `fizzy_find_exported_function`, that operates on instance and would be convenient, when you need to get an exported function and import it into another instance. With it there's going to be 4 similar `fizzy_find_exported_*` as in C++ API.
(However it has some complications with managing the lifetime of capture, so I'm not completely sure it's a good idea)
The function operating on module is renamed there to `fizzy_find_exported_function_index`, which better reflects what it does.